lib/clock.dart

```diff
@@ -12,164 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import 'src/utils.dart';
+export 'src/clock.dart';
+export 'src/default.dart';
 
 /// Returns current time.
 @Deprecated("Pass around an instance of Clock instead.")
 typedef DateTime TimeFunction();
 
-/// Return current system time.
+/// Returns the current system time.
 @Deprecated("Use new DateTime.now() instead.")
 DateTime systemTime() => new DateTime.now();
 
-/// A provider for the "current time" and points relative to the current time.
-///
-/// This class is designed with testability in mind. The current point in time
-/// (or [now()]) is defined by a function that returns a [DateTime]. By
-/// supplying your own time function or using [new Clock.fixed], you can control
-/// exactly what time a [Clock] returns and base your test expectations on that.
-class Clock {
-  /// The function that's called to determine this clock's notion of the current
-  /// time.
-  final DateTime Function() _time;
+/// Returns the current time as reported by [clock].
+@Deprecated("Use clock.now() instead.")
```

Comment:
Do we want to release < 1.0.0 so we can clean these up first or are we going to leave the deprecated stuff in for a bit?

Reply:
I think it's fine to release as 1.0.0. I generally think of deprecations as meaning "we'll remove these next time we need to do a breaking release for some other reason" unless they're particularly difficult to maintain or they're blocking other better APIs.